plugins/hooks/routes/crash.js

```diff
@@ -0,0 +1,36 @@
+'use strict';
+
+const Boom = require('boom');
+const Joi = require('joi');
+
+const controllerFunc = require('../controller');
+
+module.exports = {
+  path: '/v1/hooks/crash',
+  method: 'POST',
+  config: {
+    description: 'Webhook endpoint for the crash event',
+    tags: ['api', 'v1', 'crash'],
+    response: {
+      schema: Joi.object({ status: 'OK' })
+    }
+  },
+  handler: (req, reply) => {
+    const caller = req.payload.project.domain;
+    const authClient = req.server.plugins['authentic-client'].client;
+    const controller = controllerFunc(authClient, req);
+    const principalId = req.payload.project.name;
+    const keyId = req.payload.project.id;
```

Comment:
minor note: semantically... should we prefix the keyId with "modulus-" (that way if looking in the authentic keystore they would be more meaningful);  this is still an issue for multiple-servos... if that is out-of-scope this time then so be it; but should definitely be addressed soon.
on that note, the reading of these 2 common webhook values to authentic terms is copy/pasted in each route... if there were more i'd definitely say we need a helper module... but this is just small enough i'd say you could go either way.